transit_odp/pipelines/tests/test_dataset_etl/test_etl_serviced_organisations.py

```diff
@@ -0,0 +1,203 @@
+from dateutil import tz
+import pandas as pd
+
+from transit_odp.pipelines.tests.test_dataset_etl.test_extract_metadata import (
+    ExtractBaseTestCase,
+)
+from transit_odp.pipelines.tests.utils import check_frame_equal
+from transit_odp.transmodel.models import ServicedOrganisations
+
+TZ = tz.gettz("Europe/London")
+
+
+class ETLServicedOrganisations(ExtractBaseTestCase):
+    test_file = (
+        "data/test_serviced_organisations/test_extract_serviced_organisations.xml"
+    )
+
+    def test_extract(self):
+        # setup
+        file_id = hash(self.file_obj.file)
+
+        # test
+        extracted = self.xml_file_parser._extract(self.doc, self.file_obj)
+
+        expected_serviced_organisation = pd.DataFrame(
+            [
+                {
+                    "file_id": file_id,
+                    "serviced_org_ref": "NYCCSC",
+                    "name": "NYCC Schools",
+                },
+            ]
+        ).set_index("file_id")
+
+        self.assertTrue(
+            check_frame_equal(
+                extracted.serviced_organisations, expected_serviced_organisation
+            )
+        )
+
+        self.assertCountEqual(
+            list(extracted.serviced_organisations.columns),
+            list(expected_serviced_organisation.columns),
+        )
+        self.assertEqual(extracted.serviced_organisations.index.names, ["file_id"])
```

Comment:
The `check_frame_equal` function matches the indexes too - so this line is not required

Reply:
Done